pages/blog/_posts/vercel-cloudflare-wait-until.mdx

```diff
@@ -0,0 +1,126 @@
+---
+heading: When to use Vercel and Cloudflare's waitUntil utility
+subtitle: What is it, when to use it, and when not to use it
+image: /assets/blog/vercel-cloudflare-wait-until/featured-image.png
```

Comment:
I like the code as a picture. What do you think about making the font bigger and maybe adding some short title or question? Just so it will be contextualized better.

(I can do it)